forest/cli/src/cli/snapshot_cmd.rs

```diff
@@ -67,6 +70,12 @@ pub enum SnapshotCommands {
         /// Use [`aria2`](https://aria2.github.io/) for downloading, default is false. Requires `aria2c` in PATH.
         #[arg(long)]
         aria2: bool,
+        /// Maximum number of times to retry the fetch
+        #[arg(short, long, default_value = "3")]
+        max_retries: Option<i32>,
+        /// Duration to wait between the retries in seconds
+        #[arg(short, long, default_value = "60")]
+        delay: Option<u64>,
```

Comment:
Could we use a strongly typed `Duration` instead?

Reply:
@LesnyRumcajs i didn't find a way to add a custom parser inside #[arg()]

Reply:
https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=71cbf458c2ce3e21fa6312d8ace3f1db

The parsing method could be put somewhere in `utils`, so it can be re-used.

Reply:
Even better, there's a crate so I believe we could just hook it up to the parser https://docs.rs/parse_duration/latest/parse_duration/